scripts/update.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+## To package a new runtime.zip from source code, run
+## zip -r runtime.zip runtime
+## zip -r runtime.zip -d "runtime/.git/*"
+
+set -e
+
+UPDATE_ZIP='/tmp/runtime.zip'
+DEST='/home/pi/test'
+
+while [ ! -f $UPDATE_ZIP ]
+do
+  sleep 2
+done
+
+echo 'IMPORTANT: Runtime Rebooting for Update' >> /tmp/log-fifo
+sleep 2
+
+SERVICES="executor dev_handler net_handler shm_stop shm_start"
+
+cd systemd && sudo systemctl stop $SERVICES
+
+unzip -o /tmp/runtime.zip -d ~/
```

Comment:
The only question I have here is like did you test to make sure this overwrites the existing runtime folder with no problems (does it ask for permission to overwrite the existing directory?) and on reboot it rebuilds Runtime? Like where/when is it rebuilding Runtime to get the updated executables?

Reply:
yes it overwrites the existing folder. it doesn't need to rebuild since the zip contains the built binaries

Reply:
@AVSurfer123  wait how does the zip contain the built binaries? bc the .zip from the repo doesn't contain built binaries right? Unless Dawn scp's the .zip, unzips it, and then builds it?

Reply:
sorry for the late reply

Reply:
oh, is the zip going to be the one directly from the github releases? I thought we could just host a zip of our working pi on the website for them to download. usually source code would be better, but every pi should be the exact same OS image here.

Reply:
I'm not really sure, but I'm pretty sure we're not gonna post the entire raspi disk image on the website...

Reply:
I'm not saying to post the raspi disk image. I said its the ZIP of the runtime folder on a working raspi